docs/rfc/active/business_glossary/README.md

```diff
@@ -0,0 +1,293 @@
+
+### BusinessTermAspect
+There will be new aspect defined to capture the required attributes & ownership information
+
+```
+/**
+ * A union of all supported metadata aspects for a BusinessTerm
+ */
+typeref BusinessTermAspect = union[
+  BusinessTermProperties,
+  Ownership
+]
+```
+
+Business Term Entity Definition
+```
+/**
+ * Data model for a Business Term entity
+ */
+record BusinessTermEntity includes BaseEntity {
+
+  /**
+   * Urn for the dataset
+   */
+  urn: BusinessTermUrn
+
+  /**
+   * Business Term native name e.g. CashInstrument
+   */
+  name: optional string
+
+}
+```
+
+### Entity BusinessTermProperties
+
+```
+/**
+ * Properties associated with a BusinessTerm
+ */
+record BusinessTermProperties {
+
+  /**
+   * Friendly Name of business term
+   */
+  name: string
+
+  /**
+   * Definition of business term
+   */
+  definition: string
+
+  /**
+   * Source of the Business Term (INTERNAL or EXTERNAL) with default value as INTERNAL
+   */
+  termSource: EnumType
+
+  /**
+   * External Reference to the business-term (URL)
+   */
+  sourceRef: optional string
+
+ /**
+   * URI of the external source when the term is borrowed from external
+   * e.g: fibo-fbc-fi-fi:CashInstrument
+   */
+  sourceURI: optional string
+
+  /**
+   * The abstracted URI such as https://spec.edmcouncil.org/fibo/ontology/FBC/FinancialInstruments/FinancialInstruments/CashInstrument.
+   */
+  uri: optional Uri
+
+}
+
+```
+
+### Business Term Realationship 
+Business Terms will be owened by certain business users
+
+```
+/**
+ * A generic model for the Owned-By relationship
+ */
+@pairings = [ {
+  "destination" : "com.linkedin.common.urn.CorpuserUrn",
+  "source" : "com.linkedin.common.urn.BusinessTermUrn"
+}, {
+   "destination" : "com.linkedin.common.urn.CorpuserUrn",
+   "source" : "com.linkedin.common.urn.BusinessTermUrn"
+} ]
+record OwnedBy includes BaseRelationship {
+
+  /**
+   * The type of the ownership
+   */
+  type: OwnershipType
+}
+```
+
+### Extension to SchemaField entity
+
+Proposing to replace the fieldPath with urn
+
+```
+record SchemaField {
+
+  /**
+   * Urn for the Dataset Field
+   */
+  urn: DatasetFieldUrn
+
+  /**
+   * Flattened name of a field in JSON Path notation.
+   */
+  jsonPath: optional string
+
+  /**
+   * Indicates if this field is optional or nullable
+   */
+  nullable: boolean = false
+
+  /**
+   * Description
+   */
+  description: optional string
+
+  /**
+   * Platform independent field type of the field.
+   */
+  type: SchemaFieldDataType
+
+  /**
+   * The native type of the field in the dataset's platform as declared by platform schema.
+   */
+  nativeDataType: string
+
+  /**
+   * There are use cases when a field in type B references type A. A field in A references field of type B. In such cases, we will mark the first field as recursive.
+   */
+  recursive: boolean = false
+
+  /**
+   * The Urn of the business term to be linked (optional)
+   */
+  businessTerm: optional Urn  
```

Comment:
Now, businessTerm is a property of a field,  I would suggest to move this into an aspect DatasetField entity. 
Let us no overload `SchemaField`.

Reply:
Might need your help here, how we will have to relate DatasetField entity in the Dataset aspect model

Reply:
done